src/pages/requests/new.tsx

```diff
@@ -0,0 +1,122 @@
+import { useState, ReactNode } from 'react';
+import Head from 'next/head';
+import {
+  Heading,
+  HeadingLevels,
+  Button,
+  ErrorMessage,
+} from 'lbh-frontend-react';
+import Layout from '../../components/DashboardLayout';
+import Field from '../../components/Field';
+import Checkbox from '../../components/Checkbox';
+import Radio from '../../components/Radio';
+import { Formik, Form } from 'formik';
+import * as Yup from 'yup';
+import fetch from 'isomorphic-unfetch';
+import evidenceTypes from './_evidence-types.json';
+
+const schema = Yup.object().shape({
+  name: Yup.string().required("Please enter the resident's name"),
+  email: Yup.string()
+    .required("Please enter the resident's email address")
+    .email('Please give a valid email address'),
+  phone: Yup.string().required("Please enter the resident's phone number"),
+  sendByEmail: Yup.boolean(),
+  sendBySms: Yup.boolean(),
+  evidence: Yup.string().required('Please choose an evidence type'),
+});
+
+const SendARequest = (): ReactNode => {
+  const [submitError, setSubmitError] = useState(false);
+
+  return (
+    <Layout>
+      <Head>
+        <title>Make a new request</title>
+      </Head>
+      <Heading level={HeadingLevels.H2}>Make a new request</Heading>
+
+      <Formik
```

Comment:
It might make things easier to test if we pull this form into a separate component, along with the yup schema (e.g. `NewRequestform` or whatever), that way our page components stay super thin and easy to grok

Reply:
sounds good